libs/messaging/kafka-message-bus-impl/src/main/kotlin/net/corda/messagebus/kafka/config/MessageBusConfigResolver.kt

```diff
@@ -66,6 +68,10 @@ internal class MessageBusConfigResolver(private val smartConfigFactory: SmartCon
         // Trim down to just the Kafka config for the specified role.
         val roleConfig = resolvedConfig.getConfig("roles.$rolePath")
         val properties = roleConfig.toKafkaProperties()
+
+        //enforce the partitioner to be our custom partitioner. Kafka Config requires it to be of type CLASS
+        properties[PARTITIONER_CLASS_CONFIG] = KafkaProducerPartitioner().javaClass
```

Comment:
Do we need to create an instance here?
```suggestion
        properties[PARTITIONER_CLASS_CONFIG] = KafkaProducerPartitioner::class.java
```

Reply:
thats better, thanks